.github/workflows/continuous-integration.yml

```diff
@@ -312,11 +366,34 @@ jobs:
             .cache/yarn
             node_modules
 
+      - name: List changed files
+        id: changed-files
+        run: echo ::set-output name=all_changed_files::$(git diff-tree --no-commit-id --name-only -r ${{ github.sha }})
+
+      - name: Get app folders for changed files
+        id: get-changed-apps
+        run: echo ::set-output name=app_folders::$(node script/github-actions/get-changed-apps.js --output-type folder | tr ',' ' ')
```

Comment:
Since we're doing this in a few places now, I'm probably going to add a `delimiter` option for this script, so we won't need to transform the output in the workflow